rancid-tomatillos/src/Login/Login.js

```diff
@@ -1,13 +1,12 @@
 import React from "react";
-import {BrowserRouter as Router, Route} from "react-router-dom";
-import MainView from "./MainView";
 
 class Login extends React.Component {
     constructor() {
         super();
         this.state = {
             email: "",
             password: "",
+            badLogin: false,
```

Comment:
This is something to look into more -- do we wanna follow the same patterns with using `App.state.loggedIn` or is there a preferred way of handling wrong username/PW?